cpp/bench/CMakeLists.txt

```diff
@@ -63,7 +64,7 @@ if(BUILD_CUML_PRIMS_BENCH)
 
   add_dependencies(prims_benchmark spdlog)
 
-  target_link_libraries(prims_benchmark benchmarklib)
+  target_link_libraries(prims_benchmark ${CUDA_cublas_LIBRARY} benchmarklib)
```

Comment:
Not sure if this is the right place to add this dependency.

Reply:
Yes, this should be fine

Reply:
Should we link cublas only, or `CUML_LINK_LIBRARIES`? For prims tests we do the latter https://github.com/rapidsai/cuml/blob/51c90b260ac49da0445f5c3665d30750614bc864/cpp/test/CMakeLists.txt#L191

Reply:
Currently prims benchmark is in a very early stage. So, `CUML_LINK_LIBRARIES` might be an overkill. Later when we start to use many other libs, we can add this one instead.